Pythonwin/pywin/framework/app.py

```diff
@@ -354,7 +354,9 @@ def OnInitDialog(self):
         except OSError:
             ver = None
         if not ver:
-            warnings.warn(f"Could not read pywin32's version from '{version_path}'")
```

Comment:
I don't think this is correct in practice - the caller here isn't going to be python code.